client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -230,6 +230,11 @@ class NotificationSubscriptions extends Component {
 									) }
 								</span>
 							</FormLabel>
+							<FormSettingExplanation>
+								{ this.props.translate(
```

Comment:
@arcangelini Can we make the copy conditional so that it can ship sooner? If we have the translation show it, if not show nothing?